ui/src/dashboards/actions/index.ts

```diff
@@ -362,6 +363,8 @@ export const getDashboardAsync = (dashboardID: string) => async (
 
     dispatch(setViews(RemoteDataState.Done, views))
 
+    dispatch(getChecks())
```

Comment:
Is it OK for a dashboard to render before these are loaded? Or should this be inserted right after line 356?